applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,1714 @@
+{
```

Comment:
<p>"Below is a chart of WBGT from relative humidity and temperature:" </p><p>Is the chart missing? It moves from this line to "This method tends to overpredict WBGT..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Huh it isn't showing up on the builds, not sure why. Thanks @cyschneck